l10n_ch_hr_payroll/__openerp__.py

```diff
@@ -26,23 +26,42 @@
 #
 ##############################################################################
 
+
 {
     'name': 'Switzerland - Payroll',
     'summary': 'Swizerland Payroll Rules',
     'category': 'Localization',
     'author': "Open-Net Sàrl,Odoo Community Association (OCA)",
+    'description': """
```

Comment:
Please remove description from `__openerp__.py`

`README.rst` will be displayed in Odoo and on apps.odoo.com only if `description` key is not present in `__openerp__.py`